core/utxo/system_call.go

```diff
@@ -42,16 +41,15 @@ func (uv *UtxoVM) SystemCall(contractName, methodName string, args map[string][]
 	if err != nil {
 		return nil, lastConfirmedTime, lastConfirmedHeight, err
 	}
+
+	//vData, err := reader.Get(contractName, []byte(methodName))
 	invokeRes, invokeErr := ctx.Invoke(methodName, args)
 	if invokeErr != nil {
 		ctx.Release()
 		return nil, lastConfirmedTime, lastConfirmedHeight, invokeErr
 	}
 	rset, _, _ := modelCache.GetRWSets()
 	ctx.Release()
-	if !withConfirmed {
-		return invokeRes.Body, lastConfirmedTime, lastConfirmedHeight, nil
-	}
 	for _, v := range rset {
```

Comment:
This no more use while use snapshot reader.

Reply:
Done :)